docs/README.md

```diff
@@ -39,19 +58,24 @@ The Venice write path can be broken down into three granularities: full dataset
 | Updates to some columns of some rows            | Incremental Push Job doing Write Compute | Real-Time Job doing Write Compute |
 
 ### Hybrid Stores
-Moreover, the three granularities of write operations can all be mixed within a single dataset. A dataset which gets full dataset swaps in addition to row insertion or row updates is called _hybrid_.
+Moreover, the three granularities of write operations can all be mixed within a single dataset. A dataset which gets 
+full dataset swaps in addition to row insertion or row updates is called _hybrid_.
 
-As part of configuring a store to be _hybrid_, an important concept is the _rewind time_, which defines how far back should recent real-time writes be rewound and applied on top of the new generation of the dataset getting swapped in.
+As part of configuring a store to be _hybrid_, an important concept is the _rewind time_, which defines how far back 
+should recent real-time writes be rewound and applied on top of the new generation of the dataset getting swapped in.
 
-Leveraging this mechanism, it is possible to overlay the output of a stream processing job on top of that of a batch job. If using partial updates, then it is possible to have some of the columns be updated in real-time and some in batch, and these two sets of columns can either overlap or be disjoint, as desired.
+Leveraging this mechanism, it is possible to overlay the output of a stream processing job on top of that of a batch 
+job. If using partial updates, then it is possible to have some of the columns be updated in real-time and some in 
+batch, and these two sets of columns can either overlap or be disjoint, as desired.
 
 ### Write Compute
 Write Compute includes two kinds of operations, which can be performed on the value associated with a given key:
 
 - **Partial update**: set the content of a field within the value.
 - **Collection merging**: add or remove entries in a set or map.  
 
-N.B.: Currently, write compute is only supported in conjunction with active-passive replication. Support for active-actice replication is under development. 
+N.B.: Currently, write compute is only supported in conjunction with active-passive replication. Support for 
+active-active replication is under development. 
```

Comment:
AA/WC is GA now. Maybe we can update this? @sixpluszero

Reply:
We are still squashing bugs in it so I would rather leave this for a future announcement...